src/main/java/com/example/provider/json/SerializedTrip.java

```diff
@@ -33,6 +35,10 @@ abstract class SerializedTrip {
   abstract ImmutableList<Waypoint> waypoints();
   /** ID of the vehicle. */
   abstract String vehicleId();
+  /** Routes to waypoints */
```

Comment:
Let's update this comment to be more reflective of the field. Maybe something like "Route for this trip"